sc2reader/factories/plugins/replay.py

```diff
@@ -100,21 +100,30 @@ def APMTracker(replay):
     above actions divided by the number of seconds played by the player (not
     necessarily the whole game) multiplied by 60.
     """
+
+    speed_multiplier = 1
+    if replay.expansion == 'LotV':
+        speed_multiplier = 1.4
+
+    game_seconds_per_second = 1.4
+    if replay.expansion == 'LotV':
+        game_seconds_per_second = 1.4
```

Comment:
We have two issues now:
1) is/was your new test affected by whether the number was 1.0 or 1.4?  i.e. does the test test what it's supposed to test?
2) how should APMs be handled / verified

Regarding 2), this PR should either not change APMs, or it should change APMs to get close to what people see in-game.

So, two ways to go:
a) run the old pre-this-PR code on this replay, see what APM comes out, and verify that this PR does not change the APM using your test
b) run the replay in the game client, see what APM is reported and see that our computed APM is in the same neighborhood

I'm happy either way.   APM handling may be in a bad state now, but the point of this PR wasnt to fix everything about APM, but just to get the current chart to show up fully.  But we shouldn't let an unintentional change to APM logic go through unless the change moves us closer to correctness.